### PR TITLE
Update UI

### PR DESCRIPTION
**Pull request by Woozl**
image

### CODE REVIEW

File: src/components/Examples.vue [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<script setup lang="ts">
+import { exampleQueries } from '@/modules/exampleQueries';
+import { queryProvivderKey } from '@/stores/query';
```
Comment (balhoff) [inline src/components/Examples.vue:3]: Typo?

**Review by balhoff — CHANGES_REQUESTED** [review]
This looks good! Just one little typo. 😄

**Comment by balhoff** [comment]
Looks like we have a small conflict due to a couple new example queries.

**Comment by balhoff** [comment]
@Woozl are you okay with a squash merge for this PR? Or do you think we should try to preserve your commit history? I prefer squash in general but this is a big PR.

File: src/components/Examples.vue (hunk 1 above)
Comment (Woozl) [reply src/components/Examples.vue]: Yep, sometimes the autocomplete works too well :)

**Comment by Woozl** [comment]
> Looks like we have a small conflict due to a couple new example queries.

Yeah I think since I renamed that file from `queries.ts` to `exampleQueries.ts`

> @Woozl are you okay with a squash merge for this PR? Or do you think we should try to preserve your commit history? I prefer squash in general but this is a big PR.

I think a squash merge would be fine. What I'll probably do is merge `main` -> `update/ui` to fix the conflict, then squash merge this branch back into main. Is it okay if I also run an automated formatter ([Prettier](https://prettier.io/)) before the squash merge?

**Review by balhoff — APPROVED** [review]

**Pull request merged**